lib/ramble/docs/tutorials/running_an_experiment.rst

```diff
@@ -0,0 +1,270 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _running_an_experiment_tutorial:
+
+======================
+Running An Experiment
+======================
+
+This tutorial will provide a basic introduction to navigating Ramble and running
+experiments. In this tutorial, you will set up and run a benchmark simulation
+using `GROMACS <https://www.gromacs.org/>`_, a free and open-source application
+for molecular dynamics.
+
+------------
+Installation
+------------
+
+To install Ramble, see the :doc:`../getting_started` guide.
+
+----------------------
+Available Applications
+----------------------
+
+Once Ramble is installed, viewing the available application definitions is as
+simple as executing:
+
+.. code-block:: console
+
+    $ ramble list
+
+The ``ramble list`` command also takes a query string, which can be used to
+filter available application definitions. For example:
+
+.. code-block:: console
+
+    $ ramble list hp*
+
+might output the following:
+
+.. code-block:: console
+
+    ==> 3 applications
+    hpcc  hpcg  hpl
+
+Additionally, applications can be filtered by their tags, e.g.
+
+.. code-block:: console
+
+    $ ramble list -t molecular-dynamics
+    ==> 4 applications
+    gromacs  hmmer  lammps  namd
+
+The available tags (and their mappings to applications) can be listed using:
+
+.. code-block:: console
+
+    $ ramble attributes --all --tags
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+What's in an application?
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Knowing what applications are available is only part of how a user interacts
```

Comment:
I'll try to generally apply this comment, but I think we can change `user` to a more active `you` in text like this.

i.e. this is a tutorial, so we can directly tell people that they should do things.